sources/packages/web/src/components/common/notes/Notes.vue

```diff
@@ -65,19 +70,21 @@
       ref="createNotesModal"
       :entityType="entityType"
       @submitData="emitToParent"
+      :allowedRole="allowedRole"
```

Comment:
This is not required if we exclude handling the modals.

Reply:
same as above comment